app/presenters/wpcc/presenter.rb

```diff
@@ -11,9 +11,14 @@ def initialize(object, args = {})
     end
 
     def salary_frequency_options
-      Wpcc::YourDetailsForm::SALARY_FREQUENCIES.map do |frequency|
-        [text_for('salary_frequency', frequency), frequency]
-      end
+      Wpcc::SalaryFrequencyConverter::SALARY_FREQUENCIES
+        .map do |frequency, frequency_number|
+          [
+            text_for('salary_frequency', frequency),
+            frequency,
+            data: { unitconverter: frequency_number }
```

Comment:
ruby convention is to use snake_case for variable/attribute names e.g. unit_converter